packages/terra-form-input/src/terra-dev-site/doc/example/controlled/examplestyles.scss

```diff
@@ -0,0 +1,5 @@
+:local {
```

Comment:
If this file is still needed after making the button changes the file name needs renamed. If the file is not shared with other examples it should be renamed to `InputProgrammaticSet.module.scss` based on the example file name and previous naming standards.

Reply:
Renamed the file to `InputProgrammaticSet.module.scss`